templates/tlog-forward.conf.epp

```diff
@@ -0,0 +1,42 @@
+# This file is managed by Puppet. Manual changes will be lost
```

Comment:
Passing parameters here explicitly will make template easier to read and maintain in the future.
Also removes tight linkage to the puppet code, allowing the puppet code to change independent of this template and vice-versa.

Reply:
Template updated to accept passed params